python/cugraph/utilities/nx_factory.py

```diff
@@ -35,6 +36,11 @@ def convert_from_nx(nxG, weight=None):
         raise ValueError("nxG does not appear to be a NetworkX graph type")
 
     pdf = nx.to_pandas_edgelist(nxG)
+    if pdf["source"].dtype not in [np.int32, np.int64] or \
```

Comment:
Would this operation be faster if done to the `gdf` instead?

Reply:
answer: maybe, but typical inputs to this function are small enough that the difference isn't significant in practice.